scripts/cmake-configurator/parser/cmake_row.py

```diff
@@ -32,33 +35,33 @@ def __init__(self, raw_row: str, row_number: int):
       """
       row  = raw_row.strip()
       matches: Optional[Match[str]]  = re.match(self.regex_expression, row)
-      self.type: str =  self.NOT_PARSABLE
+      self.type: str =  types.NOT_PARSABLE.value
       self.val: str = ""
       self.name: str = ""
       self.var_type: str = ""
```

Comment:
**BAD_STYLE:** `var_type` should NOT be a string. Unless we're talking about very rare cases where there can be an unlimited amount of types coming from the outside world, types should be represented with Enums.